config/configuration.json

```diff
@@ -271,6 +271,11 @@
   },
   "EdgeCrossingEdgeCheck": {
     "minimum.highway.type": "no",
+    "maximum.highway.type": "trunk",
```

Comment:
Why is this set lower than the default?

Reply:
changed to default corridor from "Motorway" to "No"